jdbc/src/main/java/org/apache/metamodel/jdbc/JdbcDataContext.java

```diff
@@ -903,4 +905,17 @@ public String getDatabaseProductName() {
     public String getDatabaseVersion() {
         return _databaseVersion;
     }
+
+    public static int getSystemPropertyValue(String property, int defaultValue) {
```

Comment:
We should probably make this protected scope. I don't think we should expose this method outside of the JDBC module.